lib/network/modules/components/edges/BezierEdgeDynamic.js

```diff
@@ -125,7 +124,7 @@ class BezierEdgeDynamic extends BezierEdgeBase {
    * Draw a line between two nodes
    * @param {CanvasRenderingContext2D} ctx
    * @param {{toArrow: boolean, toArrowScale: (allOptions.edges.arrows.to.scaleFactor|{number}|allOptions.edges.arrows.middle.scaleFactor|allOptions.edges.arrows.from.scaleFactor|Array|number), toArrowType: *, middleArrow: boolean, middleArrowScale: (number|allOptions.edges.arrows.middle.scaleFactor|{number}|Array), middleArrowType: (allOptions.edges.arrows.middle.type|{string}|string|*), fromArrow: boolean, fromArrowScale: (allOptions.edges.arrows.to.scaleFactor|{number}|allOptions.edges.arrows.middle.scaleFactor|allOptions.edges.arrows.from.scaleFactor|Array|number), fromArrowType: *, arrowStrikethrough: (*|boolean|allOptions.edges.arrowStrikethrough|{boolean}), color: undefined, inheritsColor: (string|string|string|allOptions.edges.color.inherit|{string, boolean}|Array|*), opacity: *, hidden: *, length: *, shadow: *, shadowColor: *, shadowSize: *, shadowX: *, shadowY: *, dashes: (*|boolean|Array|allOptions.edges.dashes|{boolean, array}), width: *}} values
-   * @param {vis.Node} viaNode
+   * @param {Node} viaNode
```

Comment:
Urgh. Can you please point me to the reference that says this is valid?

Reply:
What's invalid about this?

Reply:
I mean (old) line 127, I'm just reacting to the sheer size of it. A typedef for this would be in order for now.